test/application.test.js

```diff
@@ -128,21 +128,25 @@ describe('Feathers application', function () {
     var app = feathers()
       .configure(feathers.rest())
       .use('/todo', function (req, res, next) {
-        req.feathers.stuff = 'custom middleware';
+        req.feathers.preService = 'pre-service middleware';
         next();
-      }, todoService)
+      }, todoService, function (req, res, next) {
+        res.set('post-service', res.data.id);
```

Comment:
I see! I did it this way because I wanted to actually ensure that the post-service handler actually runs after the service, but I can change it if you like.